pubpub/pubpub
# Superadmin dash and basic landing page management

This PR is in service of the new landing page, which is shaping up nicely. It makes these additions:

1. A new `isSuperAdmin` field on `User`. We'll want to set this to `true` for all of our personal accounts as well as `pubpub-team`. We should never create a way to manipulate this field without direct access to our database. This replaces our hard-coded check for the `pubpub-team` UUID.
2. A new dashboard at `/superadmin` where we can put stuff for superadmins, i.e. us!
3. A tab on that dashboard that lets you manipulate the lists of Pubs and Communities that will appear on the landing page. These are collectively called `LandingPageFeatures` and there's a new model for them.

It looks like this — I'm nothing without `communityAccentDark`!

![image](https://user-images.githubusercontent.com/2208769/190469846-a7846632-1ba8-4754-890a-65c10598981d.png)

_Test plan:_

- Run the unit tests:

```
npm run test-dev server/landingPageFeature
```

- Visit `/superadmin` and make sure you can add, reorganize, and remove Pubs and Communities
- Visit the homepage (you'll need to select "Base PubPub" from the Community switcher) and make sure your changes are reflected there.
